storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst

```diff
@@ -4,56 +4,92 @@
+These three files are all you need to import the table into a server running
+|Percona Server| with |XtraDB| or |MySQL| 5.6.
 
-.. note:: 
+.. note::
 
-  |MySQL| uses ``.cfg`` file which contains |InnoDB| dictionary dump in special format. This format is different from the ``.exp`` one which is used in |XtraDB| for the same purpose. Strictly speaking, a ``.cfg`` file is not required to import a tablespace to |MySQL| 5.6 or |Percona Server| 5.6. A tablespace will be imported successfully even if it is from another server, but |InnoDB| will do schema validation if the corresponding ``.cfg`` file is present in the same directory.
+  |MySQL| uses :file:`.cfg` file which contains |InnoDB| dictionary dump in
+  special format. This format is different from the :file:`.exp`` one which is
+  used in |XtraDB| for the same purpose. Strictly speaking, a :file:`.cfg``
+  file is not required to import a tablespace to |MySQL| 5.6 or |Percona
+  Server| 5.6. A tablespace will be imported successfully even if it is from
+  another server, but |InnoDB| will do schema validation if the corresponding
+  :file:`.cfg` file is present in the same directory.
 
 Importing the Table
 ===================
 
-On the destination server running |Percona Server| with |XtraDB| and `innodb_import_table_from_xtrabackup <http://www.percona.com/doc/percona-server/5.5/management/innodb_expand_import.html#innodb_import_table_from_xtrabackup>`_ option enabled, or |MySQL| 5.6, create a table with the same structure, and then perform the following steps:
+On the destination server running |Percona Server| with |XtraDB| and
+`innodb_import_table_from_xtrabackup <http://www.percona.com/doc/percona-server/5.5/management/innodb_expand_import.html#innodb_import_table_from_xtrabackup>`_
+option enabled, or |MySQL| 5.6, create a table with the same structure, and
+then perform the following steps:
 
 * Execute ``ALTER TABLE test.export_test DISCARD TABLESPACE;``
-  
-  * If you see the following message, then you must enable :term:`innodb_file_per_table` and create the table again: ``ERROR 1030 (HY000): Got error -1 from storage engine``
 
-* Copy the exported files to the ``test/`` subdirectory of the destination server's data directory 
+  * If you see the following message, then you must enable
+    :term:`innodb_file_per_table` and create the table again: ``ERROR 1030
+    (HY000): Got error -1 from storage engine``
+
+* Copy the exported files to the ``test/`` subdirectory of the destination
+  server's data directory
 
 * Execute ``ALTER TABLE test.export_test IMPORT TABLESPACE;``
 
-The table should now be imported, and you should be able to ``SELECT`` from it and see the imported data.
+The table should now be imported, and you should be able to ``SELECT`` from it
+and see the imported data.
 
+.. note::
 
+  Statistics for imported tablespace will be empty until you run the ``ANALYZE
```

Comment:
s/Statistics/Persistent statistics/. Lets also mention that they will be empty because they are stored in the system tables mysql.innodb_table_stats and mysql.innodb_index_stats and they aren't updated by server during the import. And ANALYZE TABLE will update them. Also lets give the link to the upstream bug report https://bugs.mysql.com/bug.php?id=72368.